### ISSUE TITLE
singlePage does not work

### ISSUE THREAD

**Issue by Frozen666** [gotenberg/gotenberg#1046]
See #907. The `singlePage` option does not work. Following curl results in a PDF with multiple pages:
```
curl \
--request POST http://localhost:3000/forms/chromium/convert/url \
--form url=https://www.tagesschau.de \
--form singlePage=true \
-o my.pdf 
```

**Comment by gulien** [comment]
Hello @Frozen666,

Thanks for the feedback. It looks like that on some website it does not work as expected 🤔 
But in most scenarios it works fine though. For instance:

```
curl \
--request POST http://localhost:3000/forms/chromium/convert/url \
--form url=https://en.wikipedia.org/wiki/Johannes_Gutenberg \
--form singlePage=true \
-o my.pdf 
```

**Comment by Frozen666** [comment]
Also on your suggested page I end up with a pdf of 15 pages. 
I am currently using Docker image `gotenberg/gotenberg:8` as a server.

**Comment by gulien** [comment]
This is what I get using the demo instance:

[singlePage.pdf](https://github.com/user-attachments/files/17843994/singlePage.pdf)

Which exact version of Gotenberg are you using?

**Comment by Frozen666** [comment]
You are right, it is a version issue! Specifying `gotenberg/gotenberg:8` as the image made Docker use v8.1 instead of the newest version! `gotenberg/gotenberg:8.14.1` now works with `singlePage` on the websites I tested. Thanks alot!

**Comment by hrvoj3e** [comment]
For me `singlePage` just breaks paper dimension but HTML content looks OK in PDF.

I used `paperWidth=8.27  paperHeight=11.7` and got a PDF with `8.28 × 5.51 in (landscape)`.
I landscape is on it just does the same on the other dimmension.

What is interesting, the "wrong" dimension is always close to 5 (4.9, 5.5, 5.1, ...)

Using version `8.22.0`

**Comment by gulien** [comment]
@hrvoj3e 

According to the documentation:

> If the singlePage form field is set to true, it automatically **overrides the values from the paperHeight** and nativePageRanges form fields.

IMO this is expected!

**Comment by hrvoj3e** [comment]
I misunderstood what `singlePage` option does. My bad.

**Comment by gulien** [comment]
Soon available on `edge`. Will release version `8.29.0` in the next few days.
